wolfcrypt/src/ed448.c

```diff
@@ -329,7 +371,8 @@ int wc_ed448ph_sign_msg(const byte* in, word32 inLen, byte* out, word32 *outLen,
     int  ret = 0;
     byte hash[64];
 
-    ret = wc_Shake256Hash(in, inLen, hash, sizeof(hash));
+    ret = ed448_hash(key, in, inLen, hash, sizeof hash);
```

Comment:
For code consistency please use `sizeof(hash)`.

Reply:
done